Solutions/AzureDevOpsAuditing/Hunting Queries/ADONewPackageFeedCreated.yaml

```diff
@@ -1,6 +1,8 @@
 id: 3d1aa540-b4c7-4789-8c4c-4174b3c2371f
 name: Azure DevOps - New Package Feed Created
 description: |
+  'This query identifies new Azure DevOps pipelines created by users with no prior history of pipeline creation, suggesting possible unauthorized activity and requiring verification.'
```

Comment:
This query identifies users who created new package feed to Azure DevOps pipelines, having no prior history of feed creation, suggesting possible unauthorized activity and requiring verification.